modules/flowable-cmmn-engine/src/main/java/org/flowable/cmmn/engine/impl/cmd/AbstractCaseStartEventSubscriptionCmd.java

```diff
@@ -0,0 +1,155 @@
+        for (EventPayload correlationParameter : correlationParameters) {
+            if (correlationParameter.getName().equals(correlationParameterName)) {
+                return;
+            }
+        }
+        throw new FlowableIllegalArgumentException("There is no correlation parameter with name '" + correlationParameterName + "' defined in event model "
+            + "with key '" + eventModel.getKey() + "'. You can only subscribe for an event with a combination of valid correlation parameters.");
+    }
+
+    protected CaseDefinition getLatestCaseDefinitionByKey(String caseDefinitionKey, String tenantId, CommandContext commandContext) {
+        CaseDefinitionEntityManager caseDefinitionEntityManager = CommandContextUtil.getCaseDefinitionEntityManager(commandContext);
+        CaseDefinition caseDefinition = null;
+        if (caseDefinitionKey != null && (tenantId == null || CmmnEngineConfiguration.NO_TENANT_ID.equals(tenantId))) {
+            caseDefinition = caseDefinitionEntityManager.findLatestCaseDefinitionByKey(caseDefinitionKey);
+
+            if (caseDefinition == null) {
+                throw new FlowableObjectNotFoundException("No case definition found for key '" + caseDefinitionKey + "'", CaseDefinition.class);
+            }
+
+        } else if (caseDefinitionKey != null && tenantId != null && !CmmnEngineConfiguration.NO_TENANT_ID.equals(tenantId)) {
+
+            caseDefinition = caseDefinitionEntityManager.findLatestCaseDefinitionByKeyAndTenantId(caseDefinitionKey, tenantId);
+
+            if (caseDefinition == null) {
+                CmmnEngineConfiguration caseEngineConfiguration = CommandContextUtil.getCmmnEngineConfiguration(commandContext);
+                if (caseEngineConfiguration.isFallbackToDefaultTenant()) {
+                    String defaultTenant = caseEngineConfiguration.getDefaultTenantProvider().getDefaultTenant(tenantId, ScopeTypes.BPMN, caseDefinitionKey);
+                    if (StringUtils.isNotEmpty(defaultTenant)) {
+                        caseDefinition = caseDefinitionEntityManager.findLatestCaseDefinitionByKeyAndTenantId(caseDefinitionKey, defaultTenant);
+                        
+                    } else {
+                        caseDefinition = caseDefinitionEntityManager.findLatestCaseDefinitionByKey(caseDefinitionKey);
+                    }
+                    
+                    if (caseDefinition == null) {
+                        throw new FlowableObjectNotFoundException("No case definition found for key '" + caseDefinitionKey +
+                            "'. Fallback to default tenant was also applied.", CaseDefinition.class);
+                    }
+                    
+                } else {
+                    throw new FlowableObjectNotFoundException("Case definition with key '" + caseDefinitionKey +
+                        "' and tenantId '"+ tenantId +"' was not found", CaseDefinition.class);
```

Comment:
```suggestion
                        "' and tenantId '"+ tenantId +"' was not found.", CaseDefinition.class);
```
For consistency with other messages